ghost-cat-app/src/app/Proxy/ServerFacade.ts

```diff
@@ -19,6 +21,87 @@ export class ServerFacade {
     return this.http.post<ImageQueryResponse>(this.url, imageQueryRequest);
   }
 
+  public async login(
+    userIDInput: string | null,
```

Comment:
Would you be opposed to creating a class in the Model directory that was `LoginRequest` or something similar with two attributes: userID and passwordHash? I think you could avoid creating a loginRequest variable in this method and  an `obj` variable in this method, because you would just take in a parameter of type `LoginRequest`, and then use that parameter as the second argument of the call to post below. I personally think it might look a little cleaner and help maintain some type safety because you could type the post method like this: `post<LoginRequest>(url...`.